src/Metadata/CIP25.purs

```diff
@@ -0,0 +1,261 @@
+module Metadata.CIP25
```

Comment:
Another stylistic thing: we've settled on only capitalizing the first letter of initialisms/acronyms (so it would be more consistent to use `Cip25` than `CIP25`)